api/api.go

```diff
@@ -138,15 +139,18 @@ func (s Server) Serve(ctx context.Context) (err error) {
 // ServeRPC checks for a configured RPC port, and registers a listner if so
 func (s Server) ServeRPC(ctx context.Context) {
 	cfg := s.Config()
-	if !cfg.RPC.Enabled || cfg.RPC.Port == 0 {
+	if !cfg.RPC.Enabled || cfg.RPC.Address == "" {
 		return
 	}
+	maAddress := cfg.RPC.Address
+	addr, _ := ma.NewMultiaddr(maAddress)
```

Comment:
We shouldn't ignore this error. If a user provides an improperly formatted multiaddress, it'll come up here.